soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -25,8 +26,9 @@
 PORT_REQ = PORT_LOG + 1
 BUF_SIZE = 4096
 CMD_LOG_START = "start_log"
-CMD_LOG_STOP = "stop_log"
 CMD_DOWNLOAD = "download"
+PACKET_HEADER_FORMAT_FW = 'I 42s 32s H'
```

Comment:
Add a comment with field names (and get rid of the unused `pad`?)

Reply:
Hi @marc-hb , how do you think I make it as "reserve"? Just in case there is new requirement in the future. thanks!

Reply:
You don't need to reserve anything: the only two scripts that speak this protocol both live in the same directory. If you need to change the format then you just go and do it; no need for backward compatibility. Using one script from one git version and the other script from a different git version is not supported: why would it?

Same thing when you drop all this low-level socket code and switch to something higher level :-)

Reply:
Fixed. Remove it, thanks!